landoapi/models/configuration.py

```diff
@@ -4,12 +4,19 @@
 import enum
 import logging
 
+from typing import (
+    Optional,
+    Union,
+)
+
 from landoapi.cache import cache
 from landoapi.models.base import Base
 from landoapi.storage import db
 
 logger = logging.getLogger(__name__)
 
+ConfigurationValue = Union[bool, int, str]
```

Comment:
Something about the name that makes me think this is a value and not a type. Perhaps we could just append "Type" to this.

```suggestion
ConfigurationValueType = Union[bool, int, str]
```